constants.js

```diff
@@ -80,6 +80,7 @@ var csts = {
   DEBUG                   : process.env.PM2_DEBUG || false,
   WEB_IPADDR              : process.env.PM2_API_IPADDR || '0.0.0.0',
   WEB_PORT                : parseInt(process.env.PM2_API_PORT)  || 9615,
+  WEB_STRIP_ENV_VARS      : process.env.PM2_WEB_STRIP_ENV_VARS || true,
```

Comment:
Not sure about some internal APIs, but yeah this might introduce a breaking change. I simply opted for the "secure by default" option, but we might want to set this to `false` in case it does break backwards compatibility.